stdlib/log-api/src/main/ballerina/src/log/natives.bal

```diff
@@ -67,3 +91,31 @@ public function printTrace(anydata|(function () returns (anydata)) msg) = @java:
 public function printWarn(anydata|(function () returns (anydata)) msg) = @java:Method {
     class: "org.ballerinalang.stdlib.log.Utils"
 } external;
+
+# Sets the module log level. If the module is not defined, the log level will be set to the current module.
+# Following log levels are allowed.
+#
+# ERROR - error log level
+#
+# WARN - warn log level
+#
+# INFO - info log level
+#
+# DEBUG - debug log level
+#
+# TRACE - trace log level
+#
+# OFF - turns off logging
+#
+# ALL - enables all the log levels
+#
+# ```ballerina
+# log:setModuleLevelLog(log:DEBUG, moduleName=“alpha”);
+# log:setModuleLevelLog(log:WARN);
+# ```
+#
+# + logLevel - Log level to be set
+# + moduleName - Name of the module
+public function setModuleLogLevel(LogLevel logLevel, public string moduleName = "") = @java:Method {
```

Comment:
Shall we make the second param as nilable type `public string? moduleName = ()`

Reply:
Updated in 4bbc1b7